pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -509,79 +528,87 @@ func (r *ReconcileIntrusionDetection) Reconcile(ctx context.Context, request rec
 		HasNoLicense:                 hasNoLicense,
 		TrustedCertBundle:            trustedBundle,
 		IntrusionDetectionCertSecret: intrusionDetectionKeyPair,
-		UsePSP:                       r.usePSP,
+		Namespace:                    helper.InstallNamespace(),
+		BindNamespaces:               namespaces,
+		Tenant:                       tenant,
 	}
-	comp := render.IntrusionDetection(intrusionDetectionCfg)
+	intrusionDetectionComponent := render.IntrusionDetection(intrusionDetectionCfg)
 
-	if err = imageset.ApplyImageSet(ctx, r.client, variant, comp); err != nil {
+	if err = imageset.ApplyImageSet(ctx, r.client, variant, intrusionDetectionComponent); err != nil {
```

Comment:
I have never thought about this, but at the moment, if we define an ImageSet, we apply all those images to all tenants, right ? Does this need to be per tenant ?

Reply:
Maybe, although I don't think we ever use ImageSets for multi-tenancy right now. Probably one to consider later.